site/docs/releases.md

```diff
@@ -88,6 +88,20 @@ The 1.8.1 release contains bug fixes and fixes to LICENSE/NOTICE files. For full
 
 ## Past releases
 
+### 1.7.2 release
```

Comment:
I'd probably sort this by release version, so this should go after 1.8.0

Reply:
ok, I was hesitating (using release version or release date). Let me update there.

Reply:
haha. We both commented at the same time. I agree.